.github/workflows/tutorials.yml

```diff
@@ -85,7 +85,7 @@ jobs:
       if: matrix.name != 'tutos-docker-gcc-py37'
       run: |
         python -m pip install --upgrade pip
-        pip install -e .
+        pip install --upgrade -e .
```

Comment:
In purpose to trigger the error

Reply:
That doesn't make sense, why would matplotlib be already installed? And if it is then you need the --upgrade in the line below not here.